WordPress/src/main/AndroidManifest.xml

```diff
@@ -761,12 +761,6 @@
             android:theme="@style/WordPress.NoActionBar"
             tools:ignore="LockedOrientationActivity"/>
 
-        <!-- Lib activities-->
-        <activity
-            android:name="com.yalantis.ucrop.UCropActivity"
-            android:label="@string/edit_photo_screen_title"
-            android:theme="@style/WordPress.NoActionBar"/>
-
```

Comment:
We have to remove this from the Wordpress manifest because it's also declared on the SDK manifest. To be honest, after having to do this, I think we should remove this activity declaration from the SDK and just add in the documentation that, if they want to use the upload `@Composable` they should add those lines.